src/components/schedule-page/toolbar/toolbar-editing.component.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { Button } from "../../common-components";
+import { Box } from "@material-ui/core";
+import ValidationDrawerComponent from "../validation-drawer/validation-drawer.component";
+import backend from "../../../api/backend";
+import { ScheduleErrorActionType } from "../../../state/reducers/schedule-errors.reducer";
+import { ActionModel } from "../../../state/models/action.model";
+import { ScheduleError } from "../../../common-models/schedule-error.model";
+import { useDispatch, useSelector } from "react-redux";
+import { ApplicationStateModel } from "../../../state/models/application-state.model";
+import SchedulePage from "../schedule-page.component";
+
+export function ToolbarEditingComponent(props): JSX.Element {
+  const handleEditMode = props.editModeChange;
+  const schedule = useSelector((state: ApplicationStateModel) => state.scheduleData?.present);
+  const dispatcher = useDispatch();
+
+  function toggleEdit(open: boolean): void {
+    handleEditMode(open);
+  }
+
+  async function onShowErrorsClicked(): Promise<void> {
```

Comment:
Wiem, że to nie twoja metoda, ale ja bym zmienił nazwę funkcji, bo to co ona robi to strzał do bazy, a potem update o zwrócone dane stanu aplikacji. Czyli moim zdaniem, lepiej będzie nazwać metodę `updateScheduleErrors`

Reply:
Zamieniona